### PR TITLE
Move sphinx upper pin from 7 to 8.

### LINKED ISSUE #71: Unpin sphinx when sphinx_rtd_theme updates
The 2.0 release should support 7.

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #71

### CODE REVIEW

**Comment by lilyminium** [comment]
Do we want to pin to under 8 or just leave the max pin off?

**Comment by IAlibay** [comment]
sphinx_rtd_theme is pinned at <8 but I'm generally in favour of no pins outside of releases - edit: I'll leave it to you, I can go either way here

**Comment by lilyminium** [comment]
Would changing sphinx_rtd_theme>=1.3.0 instead work better here then?

**Comment by IAlibay** [comment]
Sorry, I'm not sure I follow. I'm leaving this up to you if that's ok - I'm just going across the ecosystem and opening up PRs for things that need maintenance.

File: setup.py [hunk 1]
```diff
@@ -142,7 +142,7 @@ def copy_mda_assets():
         python_requires='>=3.9',
         install_requires=[
             'sphinx_rtd_theme>=1.2',
```
Comment (lilyminium) [inline setup.py]:
```suggestion
            'sphinx_rtd_theme>=1.3',
```

File: setup.py [hunk 2]
```diff
@@ -142,7 +142,7 @@ def copy_mda_assets():
...
-            'sphinx>=6.2.1,<7.0',
+            'sphinx>=6.2.1,<8.0',
```
Comment (lilyminium) [inline setup.py]:
```suggestion
            'sphinx>=6.2.1',
```

**Comment by lilyminium** [comment]
Ok, in that case I'll take over -- thanks for starting it.

File: setup.py (hunk 1 above)
Comment (lilyminium) [reply setup.py]: Note: 1.3 is the first version that supported sphinx 7

**Comment by lilyminium** [comment]
I'll leave this up until tomorrow my time in case you want to review, else I'll merge :-)

**Comment by IAlibay** [comment]
> I'll leave this up until tomorrow my time in case you want to review, else I'll merge :-)

In the nicest way possible, it's just pins - at least to me, as long as we achieve the "allow sphinx 7" criteria, everything is perfectly fine.

**Comment by IAlibay** [comment]
Looks like there's a bug in the yaml file, can you yank the release? I'll debug the issue.

**Comment by lilyminium** [comment]
Removed -- thanks for looking into it.

**Pull request merged**